README.md

```diff
@@ -43,6 +43,13 @@ More help is available from the Maria Discuss mailing list
 https://launchpad.net/~maria-discuss
 and the #maria IRC channel on Freenode.
 
+Live QA for beginner contributors
```

Comment:
There is actually a duplicate link above as well (https://mariadb.com/kb/en/mariadb-versus-mysql-features/ and https://mariadb.com/kb/en/mariadb-vs-mysql-features/) - one of them should be removed.

Reply:
Final pedantic comment is that titles shouldn't end in punctuation, so Help, Licence and Bug Reports should not have colons at the end ("Help" rather than "Help:"). OK to push when done.

Reply:
Done, thanks @iangilfillan .